pyproject.toml

```diff
@@ -32,8 +31,12 @@ protobuf = "4.21.12"
 Cerberus = "1.3.4"
 async-timeout = "4.0.2"
 pyln-client = "23.8"
-cashu = "0.9.0"
 slowapi = "^0.1.7"
+python-dotenv = "^1.0.0"
+websocket-client = "^1.6.1"
+secp256k1 = "^0.14.0"
+pycryptodomex = "^3.18.0"
```

Comment:
whats with those ones?

Reply:
this is a bit weird because those deps are needed in lnbits but got removed at some point and are pulled from the `cashu` package, which should actually be removed at some point.